tests/functional/test_logs.py

```diff
@@ -5,6 +5,11 @@
 from tests.helper import TestHelper
 import time
 
+WAIT_FOR_FIRSTLOGS_TIMEOUT_S = 5
```

Comment:
Please add some comments here on why these different timeouts are required and how you chose the values. This will help if we ever need to adjust in the future.

Reply:
Agree!